### PR TITLE
ISPN-9259 Fix agent-bond and jolokia module paths

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.jboss.org/browse/ISPN-9363

### CODE REVIEW

**Comment by ryanemerson** [comment]
I'll backport this to 9.3.x as well.

File: server/integration/feature-pack/src/main/resources/content/bin/jolokia.bat [hunk 1]
```diff
@@ -1,5 +1,5 @@
 set AGENT_BOND_JAR=
-for /r %JBOSS_HOME%\modules\io\fabric8\agent-bond\main %%X in (*.jar) do (
+for /r %JBOSS_HOME%modules\system\add-ons\@infinispan.module.slot.prefix@\io\fabric8\agent-bond\@infinispan.module.slot@ %%X in (*.jar) do (
```
Comment (wburns) [inline server/integration/feature-pack/src/main/resources/content/bin/jolokia.bat]: Hrmm, can we assume that `JBOSS_HOME` env variable will always have the separator present?

Comment (ryanemerson) [reply server/integration/feature-pack/src/main/resources/content/bin/jolokia.bat]: Nope, that was removed by mistake

Comment (ryanemerson) [reply server/integration/feature-pack/src/main/resources/content/bin/jolokia.bat]: Fixed

**Comment by wburns** [comment]
Looks good to me (assuming Ryan tested it with Jolokia), I can merge unless anyone else has any last second things they wanted to add.

**Comment by wburns** [comment]
Integrated into master, thanks @ryanemerson !

**Pull request merged**